pkg/testrunner/template/helper.go

```diff
@@ -15,23 +15,47 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
-func getLatestK8sVersion(gardenerKubeconfigPath, cloudprofile, cloudprovider string) (string, error) {
+// getVersions returns all K8s version that should be rendered by the chart
+func getVersions(parameters *TestrunParameters) ([]string, error) {
+	if !parameters.MakeVersionMatrix && parameters.K8sVersion != "" {
+		return []string{parameters.K8sVersion}, nil
+	}
+	// if the kubernetes version is not set, get the latest version defined by the cloudprofile
+	if !parameters.MakeVersionMatrix && parameters.K8sVersion == "" {
+		version, err := getLatestK8sVersion(parameters.GardenKubeconfigPath, parameters.Cloudprofile, parameters.Cloudprovider)
```

Comment:
```suggestion
		version, err := getLatestK8sVersion(parameters)
```
Try to reduce number of arguments if possible.

Reply:
I did not wanted to pass the whole paramters as this function seems to be usefull as a util function in the future. Therefore I think it's more descriptive to just provide the parameters needed and not all where only 3 are needed.